### PR TITLE
Support Light mode

### PR DESCRIPTION
**Pull request by kkngsm**
#33

1. supported light mode
2. added dark/ligtht switch to example

### CODE REVIEW

File: egui_node_graph/src/editor_ui.rs [hunk 1]
```diff
@@ -599,12 +610,25 @@ where
...
+            }
         } else if resp.hovered() {
-            color_from_hex("#dddddd").unwrap()
+            if dark_mode {
+                color_from_hex("#dddddd").unwrap()
+            } else {
+                color_from_hex("#222222").unwrap()
+            }
         } else {
-            color_from_hex("#aaaaaa").unwrap()
```
Comment (setzer22) [inline egui_node_graph/src/editor_ui.rs:607]: I don't think I agree with clippy on this one 🤔 feel free to drop an #[allow] in there to silence the warning.

**Review by setzer22 — APPROVED** [review]
Thanks for submitting! 😄  This looks good to me, there's just a clippy lint preventing the merge in CI, other than that we should be good to go.

File: egui_node_graph/src/editor_ui.rs (hunk 1 above)
Comment (kkngsm) [reply egui_node_graph/src/editor_ui.rs:607]: Thanks for checking it out.
I have added #[allow].

**Comment by setzer22** [comment]
Looks very nice! Thanks again :+1:

**Pull request merged**